src/Microsoft.Health.Fhir.Core/Features/Operations/Export/ExportJobResult.cs

```diff
@@ -0,0 +1,54 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using EnsureThat;
+using Hl7.Fhir.Model;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Fhir.Core.Features.Export
+{
+    /// <summary>
+    /// This class is used to hold data that needs to be returned to the client when the
+    /// export job completes. This is a subset of the data present in ExportJobRecord.
+    /// </summary>
+    public class ExportJobResult
+    {
+        public ExportJobResult(Instant transactionTime, Uri requestUri, bool requiresAccessToken, IList<ExportFileInfo> output, IList<ExportFileInfo> errors)
+        {
+            EnsureArg.IsNotNull(transactionTime, nameof(transactionTime));
+            EnsureArg.IsNotNull(requestUri, nameof(requestUri));
+            EnsureArg.IsNotNull(output, nameof(output));
+            EnsureArg.IsNotNull(errors, nameof(errors));
+
+            TransactionTime = transactionTime;
+            RequestUri = requestUri;
+            RequiresAccessToken = requiresAccessToken;
+            Output = output;
+            Errors = errors;
+        }
+
+        [JsonConstructor]
+        public ExportJobResult()
+        {
+        }
+
+        [JsonProperty("transactionTime")]
+        public Instant TransactionTime { get; }
```

Comment:
I think you need private setter. Otherwise, the deserialization won't be able to set the value correctly.

Reply:
Use the DateTimeOffset instead of Instant here.

Reply:
I didn't add the private setters here because we will not be de-serializing it on the server side. This will be the final result that we will return to the clients. I missed removing the `[JsonConstructor]` ctor and will remove that.

Regarding the TransactionTime, property the spec - https://github.com/smart-on-fhir/fhir-bulk-data-docs/blob/master/export.md - mentions we have to use a FHIR `Instant` type. Since I was not sure whether I could use `DateTimeOffset`, I went with `Instant`. Will make the change.

Reply:
Changed `TransactionTime` type to `DateTimeOffset` from `Instant`